style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
```

Comment:
- [ ] Please make sure your design matches the given Figma design as much as possible. This is regarding your body element. It has a surrounding white space that should not be placed there. So please make sure to check all the `padding` and `margin` rules and modify them if necessary to match the design as much as possible.

Reply:
@Lordkaito, Appreciate your efforts, I just changed the margin and padding, and now it's the same as Figma design.